config/checkstyle-input-suppressions.xml

```diff
@@ -597,16 +597,6 @@
             files="checks[\\/]javadoc[\\/]javadocmethod[\\/]InputJavadocMethodPublicOnly1.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]javadoc[\\/]javadocmethod[\\/]InputJavadocMethodReceiverParameter.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]javadoc[\\/]javadocmethod[\\/]InputJavadocMethodReceiverParameter.java"/>
```

Comment:
Looks like you have missed removal of suppression here. File `InputJavadocMethodReceiverParameter.java` is still being suppressed.

Reply:
@MANISH-K-07 I solved the problem but It gives me "This branch cannot be rebased due to conflicts"

Reply:
That must be because the changes you have made have already been done and merged in any other PR